requirements/production.txt

```diff
@@ -6,7 +6,7 @@
 
 # WSGI Handler
 # ------------------------------------------------
-gevent==1.4.0
+gevent==20.9.0
```

Comment:
It doesn't appear that we're actually using gevent

Reply:
https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-6f1f964e63cc3c4b164496bc75516b41L9